data/arc notes.txt

```diff
@@ -102,7 +102,7 @@ BELOW ARE NOTABLE CHOICES FOR LIMITED ARCS (OR LACK THEREOF). Unless otherwise s
 	
 	* MET PAR TEK - The Met shares the same turret structures as the KIV (the six forward turrets), so it currently shares the same limited arcs (unlike the carriers). For a ship its size, 6 turrets is decently strong, so the blindspot given in the rear by this offsets this pocket warship.
 	
-	* MODEL SHIPS - Each model ship has drastic structural obstruction and oddly-angled turret mounts (the modeled bases themselves), so a lot of soft exceptions and some decent ones have been given from ship to ship. Overall, the Model series has less forward-firing turrets (as opposed to their vanilla all turrets forward), with more arcs around the ships in varied angles. Will be keeping an eye on this, especially with ship balance vs. the Sestor ships, but in initial tests there doesn't seem to be a huge scale shift as of yet.
+	* MODEL # SHIPS - Each model ship has drastic structural obstruction and oddly-angled turret mounts (the modeled bases themselves), so a lot of soft exceptions and some decent ones have been given from ship to ship. Overall, the Model series has less forward-firing turrets (as opposed to their vanilla all turrets forward), with more arcs around the ships in varied angles. Will be keeping an eye on this, especially with ship balance vs. the Sestor ships, but in initial tests there doesn't seem to be a huge scale shift as of yet.
```

Comment:
Your choice on this, I’m not sure that it works better but Model Ships doesn’t work great either in my opinion.